lib/ruby_lsp/ruby_lsp_rails/definition.rb

```diff
@@ -85,24 +87,26 @@ def handle_callback(node)
         end
       end
 
+      sig { params(node: Prism::CallNode).void }
+      def handle_association(node)
+        association_name = T.cast(T.must(node.arguments).arguments.first, Prism::SymbolNode).unescaped
+
+        result = @client.association_target_location(
+          model_name: @nesting.join("::"),
+          association_name: association_name,
+        )
+
+        return unless result&.key?(:location)
```

Comment:
If the result is not `nil`, then location is guaranteed to be present.
```suggestion
        return unless result
```